packages/evm/src/opcodes/EIP2929.ts

```diff
@@ -29,9 +29,9 @@ export function accessAddressEIP2929(
     // CREATE, CREATE2 opcodes have the address warmed for free.
     // selfdestruct beneficiary address reads are charged an *additional* cold access
     // if verkle not activated
-    if (chargeGas && !common.isActivatedEIP(6800)) {
+    if (chargeGas && (!common.isActivatedEIP(6800) || !common.isActivatedEIP(7864))) {
```

Comment:
I don't think that's right, as one of them will always be deactivated, right? I think we might want to adjust to
```suggestion
    if (chargeGas && !(common.isActivatedEIP(6800) || common.isActivatedEIP(7864))) {
```
To skip if both are not activated (rather than skipping if one is not active)